test/penalizer/Penalizer.test.ts

```diff
@@ -290,6 +273,81 @@ contract('Penalizer', function ([relayOwner, relayWorker, relayManager, otherAcc
     })
   })
 
+  describe('should receive/reject claims according with the commitment time', function () {
+    before(async function () {
+      await penalizer.setHub(relayHub.address, { from: await penalizer.owner() })
+    })
+    beforeEach(async () => {
+      await token.mint('1000', forwarder.address)
+
+      await relayHub.stakeForAddress(relayManager, 1000, {
+        from: relayOwner,
+        value: ether('1'),
+        gasPrice: gasPrice
+      })
+      await fundAccount(fundedAccount, sender.address, '10')
+    })
+
+    // successful claims
+    const acceptableCommitmentTimes = [
+      // due to security implications, we accept a up to 15 secs of delay
+      // but we cannot set exactly this time in tests
+      10,
+      0,
+      // set a past commitment time
+      -100,
+      -150
+    ]
+    acceptableCommitmentTimes.forEach((timeDiff, index) => {
+      it(`accept them if commit time is ${Math.abs(timeDiff)} seconds in ${timeDiff < 0 ? 'past' : 'future'}`, async function () {
+        const [rr, sig] = await createRelayRequestAndSignature({ relayData: `0xdeadbeff1${index}`, enableQos: true })
+        const now = getSecondsSinceUnixEpoch()
+        const receiptTime = now + timeDiff
```

Comment:
if you were to rename the `getSecondsSinceUnixEpoch` to something like `now` or `currentTime`, you could do something like:

```typescript
const receiptTime = currentTime() + timeDiff
```

not a blocker though

Reply:
Thanks for the suggestion, I would use `currentTimeInSeconds`